module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraTrustedCertificateAuthority.md

```diff
@@ -45,14 +44,17 @@ $cer = Get-EntraTrustedCertificateAuthority #Get the CertificateAuthorityInforma
 Remove-EntraTrustedCertificateAuthority -CertificateAuthorityInformation $cer[0]
 ```
 
-```output
+```Output
```

Comment:
We need consistency.
I see `Output` starts with a capital letter then in other areas I see `powershell` starting with small letter

Reply:
That is the design: Output should always start with capital O and powershell is all small letters. That is based on the general recommendation from the core PowerShell docs team.

Reply:
The language tag is not case-sensitive, but you should use lowercase except for a few special cases.

Code fences without tags can be used for syntax blocks or other types of content where syntax highlighting is not required.

When showing output from a command, use a tagged code fence with the language tag Output. The rendered box is labeled as Output and doesn't have syntax highlighting.

- From the PowerShell docs